lib/foreground/index.js

```diff
@@ -56,9 +53,11 @@ function copyText(){
  * @param {Node} node
  */
 
-function handle(event, node){
-  var path = cssPath(node);
-  var message = [event, path];
-  if (node.value) message.push(node.value);
-  chrome.runtime.sendMessage(message);
+function handle(event, node) {
+  if (chrome && chrome.runtime) {
```

Comment:
This method sometimes throws an error of `runtime` being `undefined`. This temporarily solves the problem.